### PR TITLE
Test cleanups

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

File: src/plone/z3cform/layout.rst [hunk 1]
```diff
@@ -105,22 +104,25 @@ now make the actual call to ``wrap_form`` and register the view class
...
     ...                provides=Interface,
     ...                factory=view_class,
     ...                name=u'test-form2')
 
-Let's render this view:
+Let's render this view::
 
     >>> view = getMultiAdapter(
     ...     (context, request), name=u'test-form2')
+    >>> view.update(debug=1)
```
Comment (gforcada) [inline src/plone/z3cform/layout.rst]: This seems to be a leftover? :thinking: the Plone Jenkins CI pull request 5.2 job failed only at this point: https://jenkins.plone.org/job/pull-request-5.2/1536/

**Comment by petschki** [comment]
@jensens I've started jenkins with this pull request here https://jenkins.plone.org/job/pull-request-5.2/1538/ but do you know, why Travis is failing?

**Comment by jensens** [comment]
It used to fail before, maybe b/c it's testing with Plone 5.1, which is pinned to the 0.9.x branch and we're on master here.

**Comment by jensens** [comment]
I add this branch to the _py3.cfg_ so we can see the effect together with our other changes.

**Comment by petschki** [comment]
I've removed the skipped test here https://github.com/zopefoundation/plone.z3cform/pull/20

**Comment by petschki** [comment]
Travis get's `plone.app.testing==6.1.0` but there's maybe a related fix on master branch by @Rotonen ... maybe release 6.1.1 and test again?

**Comment by jensens** [comment]
Plone merge fest .. merges it

**Pull request merged**